src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -1,125 +1,165 @@
 package core.basesyntax;
 
+import java.time.DateTimeException;
+import java.time.Instant;
 import java.time.LocalDate;
 import java.time.LocalDateTime;
 import java.time.LocalTime;
 import java.time.OffsetDateTime;
+import java.time.ZoneId;
+import java.time.ZoneOffset;
+import java.time.format.DateTimeFormatter;
+import java.util.Locale;
 import java.util.Optional;
 
 public class JavaDateTimeApi {
+    private static final String TIME_ZONE = "+02:00";
+    private static final DateTimeFormatter DATE_TIME_FORMATTER
+            = DateTimeFormatter.ofPattern("d MMM yyyy", Locale.UK);
+    private static final DateTimeFormatter FORMAT_DATE
+            = DateTimeFormatter.ofPattern("dd MMMM yyyy HH:mm", Locale.UK);
+    private static final LocalDate PRESENT_LOCAL_DATE = LocalDate.now();
```

Comment:
will this variable show actual current time if you decide to run the same method several times?

Reply:
As i think - it won`t. So I`ll move it inside method to call it only when there will needance.